package.json

```diff
@@ -151,6 +151,7 @@
           "text-field",
           "theme",
           "toolbar",
+          "top-app-bar",
```

Comment:
What about closure whitelist?

Reply:
toolbar doesn't currently exist in closure whitelist, so I didn't add this one either.

Reply:
I think any component we are writing from scratch should aim to be in the whitelist at the get-go at this point, because it's easier to deal with incrementally than having to come back through and clean up later.